src/Arcus.Templates.WebApi/appsettings.json

```diff
@@ -1,5 +1,13 @@
 {
   //#if (CertificateAuth)
-  "CertificateSubject": "YOUR CERTIFICATE SUBJECT NAME"
+  "CertificateSubject": "YOUR CERTIFICATE SUBJECT NAME",
   //#endif
+  //#if (Serilog)
+  "Telemetry": {
+    "ApplicationInsights": {
+      "InstrumentationKey": "YOUR APPLICATION INSIGHTS APPLICATION KEY"
+    }
+  },
+  //#endif
+  "MyConfigKey":  "MyConfigValue"
```

Comment:
Do we need this?

Reply:
🤣 that was the thing I explained previously; if we have a setting that we always use, we can use that instead; but otherwise the easiest and most maintainable solution would be a demostration config value (like I said 😄 ).

Reply:
This is funny